runner/engine.go

```diff
@@ -362,31 +364,31 @@ func (e *Engine) start() {
 			// go down
 		}
 
-		// already build and run now
-		select {
-		case <-e.buildRunCh:
-			e.buildRunStopCh <- true
-		default:
-		}
+		go func() {
+			// if current app is running, stop it
+			e.stopBin()
```

Comment:
is it intentional to call this outside of the main thread, and inside this goroutine now? (previously I think it was ran in the main thread?)